routes/author/index.js

```diff
@@ -19,59 +18,64 @@ module.exports = function(req, res){
       } finally {
         book.images = images;
       }
-      if(err){
+      if (err) {
         console.log(err);
         return book;
       }
       return book;
     });
-  };
+  }
 
-  var authorId = Number(req.params[0]);
-  var authors = {};
+  const authorId = Number(req.params[0]);
+  const authors = {};
+  if (isNaN(authorId)) {
+    return res.status(404).render('error', {
+      title: 'ページが見つかりません。',
+    });
+  }
 
-  var countAuthors = function(){
-    var d = Q.defer();
-    Author.count({}, function(err, count){
+  function countAuthors() {
+    const deffered = Q.defer();
+    Author.count({}, (err, count)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).